README.md

```diff
@@ -118,6 +118,9 @@ additional details on each available environment variable.
 | `ECS_IMAGE_CLEANUP_INTERVAL` | 30m | The time interval between automated image cleanup cycles. If set to less than 10 minutes, the value is ignored. | 30m | 30m |
 | `ECS_IMAGE_MINIMUM_CLEANUP_AGE` | 30m | The minimum time interval between when an image is pulled and when it can be considered for automated image cleanup. | 1h | 1h |
 | `ECS_NUM_IMAGES_DELETE_PER_CYCLE` | 5 | The maximum number of images to delete in a single automated image cleanup cycle. If set to less than 1, the value is ignored. | 5 | 5 |
+| `ECS_NONECS_IMAGE_CLEANUP_EXCLUDE` | amazonlinux:2, amazonlinux:1 | The imageCleanupExclusionList includes both ecs and nonecs images that agent won’t cleanup. |  |  |
```

Comment:
Please have jole/eric to review readme

Reply:
readme file will keep at the next PR, need to sync with jole/eric before put PR.